components/browser/session/src/main/java/mozilla/components/browser/session/SelectionAwareSessionObserver.kt

```diff
@@ -44,6 +44,19 @@ abstract class SelectionAwareSessionObserver(
         activeSession?.register(this)
     }
 
+    /**
+     * Starts observing changes to the session matching the [sessionId].
```

Comment:
Lets add some more detailed docs since it does more than just observing changes to the selected session.

Reply:
Done